src/main/scala/com/advancedtelematic/campaigner/data/DataType.scala

```diff
@@ -35,6 +34,21 @@ object DataType {
     updatedAt: Instant
   )
 
+  final case class Update(
+                           id: UpdateId,
+                           externalId: Option[String],
+                           namespace: Namespace,
+                           name: String,
+                           description: Option[String],
+                           createdAt: Instant,
+                           updatedAt: Instant
+                         )
+
+  final case class CreateUpdate(externalId: Option[String], name: String, description: Option[String]) {
+    def mkUpdate(ns: Namespace): Update =
```

Comment:
Shall we change this to makeUpdate() or createUpdate() or something like that? mkUpdate() does not seem to be perfect.

Reply:
Can do :) But in that case I would propose to rename `mkCampaign` and `mkCampaignMetadata` as well to be consistent.
Personally I'd go for `build(Update|Campaign|CampaignMetadata>`.

Reply:
`mkSomething` is used across the whole codebase, so it is ok

Reply:
I would also go for build(Update|Campaign|CampaignMetadata). I will leave it to Jerry/Simao to decide.